### DISCUSSION TITLE
windows-2019 deprecation date

### DISCUSSION THREAD

**Discussion by quasarea** [actions/virtual-environments#5343] [General questions] [answered]
Hi, 

Seeing windows-2016 going away this month I started to wonder, how long will we keep 2019 along. Are there any dates written somewhere? 

Regards, 
J

**Comment by miketimofeev** [comment, marked as answer]
@quasarea we support 2 latest LTS versions so the deprecation won't start until the next windows version is released. Probably not early than 2025
